app/controllers/user/applications_controller.rb

```diff
@@ -0,0 +1,16 @@
+class User::ApplicationsController < ApplicationController
+	before_action :ensure_either_broker_or_ff
+
+	def index
+		if current_user.bids.exists?
+			@applied_bids = current_user.bids.all
+		end
```

Comment:
Yeah seems like we didn't need it. Just placing a condition in the view file works :slightly_smiling_face: